partner_contact_in_several_companies/models/multi_contact.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
```

Comment:
Make one file per model, naming the file as the model. As example, res_partner.py. The only exception when there's a submodel, like sale.order.line, that is also included in the main model file (sale_order.py).